src/System.Diagnostics.Tools/src/System.Diagnostics.Tools.csproj

```diff
@@ -9,12 +9,17 @@
     <AssemblyName>System.Diagnostics.Tools</AssemblyName>
```

Comment:
Yeah, I mentioned that above. I'll have to watch the mirror and have a checkin ready.